jina/jaml/helper.py

```diff
@@ -87,8 +87,20 @@ def parse_config_source(path: Union[str, TextIO, Dict],
                         allow_dict: bool = True,
                         allow_json: bool = True,
                         *args, **kwargs) -> Tuple[TextIO, Optional[str]]:
-    """ Check if the text or text stream is valid
-
+    """Check if the text or text stream is valid.
+
+    # noqa: DAR401
+    :param path: the multi-kind source of the configs.
+    :param allow_stream: flag
```

Comment:
it would be nice to explain better these params, let me see if I can provide some better definitions in a while